tox.ini

```diff
@@ -50,52 +53,103 @@ basepython = python2.7
 commands =
     coverage run {envbindir}/zope-testrunner --path=src/RestrictedPython --all {posargs}
 deps =
+    -cconstraints.txt
     .[test]
     zope.testrunner
     coverage
 
 [testenv:coverage]
 basepython = python2.7
-deps = coverage
+skip_install = true
+
+deps =
+    -cconstraints.txt
+    coverage
+
 setenv =
   COVERAGE_FILE=.coverage
 
-skip_install = true
 commands =
     coverage erase
     coverage combine
     coverage html
     coverage xml
     coverage report
 
-[testenv:isort]
-basepython = python2.7
-deps = isort
-commands =
-    isort --check-only --recursive {toxinidir}/src {toxinidir}/tests {posargs}
-
 [testenv:isort-apply]
 basepython = python2.7
-deps = isort
+skip_install = true
+deps =
+    -cconstraints.txt
+    isort
+
 commands =
     isort --apply --recursive {toxinidir}/src {toxinidir}/tests {posargs}
 
-[testenv:flake8]
-basepython = python2.7
+[lint]
+skip_install = true
+
 deps =
+    -cconstraints.txt
     flake8
+    # helper to generate HTML reports:
     flake8-html
+    # Useful flake8 plugins that are Python and Plone specific:
+    flake8-blind-except
+    flake8-coding
```

Comment:
I think this one is nonsense. It might have been necessary before Python 2.5. Newer Python 2 versions do not compile the file if there is a non ASCII character in the file. Python 3 has a default of UTF-8 not longer requiring to declare it explicitly.